src/drawers/canvasBuffer.ts

```diff
@@ -30,7 +30,7 @@ export class CanvasBuffer {
     constructor(
         public screenWidth: number,
         public screenHeight: number,
-        public devicePixelRatio = 1,
+        public devicePixelRatio = window.devicePixelRatio,
```

Comment:
this isn't enough - we should recalculate the pixelWidth/pixelHeight whenever we use it I think? See https://github.com/palantir/plottable/blob/d93bdd897c972c065da4d8f5917869ba1d61d92b/src/plots/plot.ts#L205-L205